openwebui/functions/computer_link_filter.py

```
@@ -1,584 +1,209 @@
+        base_url = self.valves.SYSTEM_PROMPT_URL or (
+            self.valves.FILE_SERVER_URL.rstrip("/") + "/system-prompt"
+        )
+        params = {}
+        if chat_id:
+            params["chat_id"] = chat_id
+        if user_email:
+            params["user_email"] = user_email
+        url = base_url + ("?" + urllib.parse.urlencode(params) if params else "")
+
+        try:
+            req = urllib.request.Request(url, method="GET")
```

Ruff (0.15.9)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)